validator/validation/readers.py

```diff
@@ -47,6 +48,10 @@ def create_reader(dataset, version):
 
     if dataset.short_name == globals.SMOS:
         reader = SMOSTs(folder_name, ioclass_kws={'read_bulk':True})
+        
+    if dataset.short_name == globals.ERA5:
+        era5_data_folder = path.join(folder_name, 'ERA5_TS')
```

Comment:
I think a simpler structure here (as for GLDAS) makes sense, instead of creating ```era5_data_folder```, we could just use ```folder_name```, which would be ```.../ECMWF_reanalysis/ERA5``` using the structure that I suggested